README.md

```diff
@@ -1,24 +1,34 @@
 # Docker for Tornado-GPU
 
 
-![](https://img.shields.io/docker/pulls/beehivelab/tornado-gpu.svg)  [![](https://img.shields.io/badge/License-Apache%202.0-orange.svg)](https://opensource.org/licenses/Apache-2.0)
+![](https://img.shields.io/docker/pulls/beehivelab/tornado-gpu.svg?color=green&label=docker%20pulls%20nvidia)  ![](https://img.shields.io/docker/pulls/beehivelab/tornado-intel-gpu.svg?color=purple&label=docker%20pulls%20intel)  [![](https://img.shields.io/badge/License-Apache%202.0-orange.svg)](https://opensource.org/licenses/Apache-2.0)
 
 ## Prerequisites
 
-The `tornado-gpu` docker image needs the docker `nvidia` daemon. 
+* The `tornado-gpu` docker image needs the docker `nvidia` daemon. 
```

Comment:
Can you split the readme into two? the first part we can explain the TornadoVM docker for NVIDIA with all instructions. The second part the one for the Intel GPU.

Reply:
Does it make sense? The only thing changes is which image to pull and the runner script.

Reply:
Users can download one of the docker images. So users don't need to go through all the prerequisites. We can divide into two.